pkg/identityserver/user_registry.go

```diff
@@ -487,8 +505,9 @@ func (is *IdentityServer) updateUserPassword(ctx context.Context, req *ttnpb.Upd
 
 var errTemporaryPasswordStillValid = errors.DefineInvalidArgument("temporary_password_still_valid", "previous temporary password still valid")
 
+var temporaryPasswordValidity = time.Hour * 24
```

Comment:
This should be (much) longer if there are two actors involved

Reply:
What do you propose for (A) self-requesting a temporary password and (B) an admin setting one?

Reply:
A day for self-requesting is acceptable, I think even an hour (the old value) is fine too. But when you do it for somebody else considering days off, holidays, time zones, departments, agendas and all the other typical corporate inefficiencies, I would say 36 or 48 hours. Having a temporary password that has been emailed around is not super secure so quick action should be taken.

AWS uses a shorter default by the way;

>By default, temporary security credentials for an IAM user are valid for a maximum of 12 hours. But you can request a duration as short as 15 minutes or as long as 36 hours using the DurationSeconds parameter. For security reasons, a token for an AWS account root user is restricted to a duration of one hour.

Reply:
So for self-requesting an hour and for admin-set 36 hours?

Reply:
Sounds good